ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Services/UserService.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace ExadelTimeTrackingSystem.BusinessLogic.Services
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using AutoMapper;
+    using ExadelTimeTrackingSystem.BusinessLogic.Services.Abstract;
+    using ExadelTimeTrackingSystem.Data.Repositories.Abstract;
+
+    public class UserService : IUserService
+    {
+        private readonly IUserRepository _repository;
+        private readonly IMapper _mapper;
+
+        public UserService(IUserRepository repository, IMapper mapper)
+        {
+            _repository = repository;
+            _mapper = mapper;
+        }
+
+        public Task<bool> ListExistsAsync(List<Guid> ids)
+        {
+            return _repository.ListExistsAsync(ids);
+        }
+
+        public Task<bool> ExistsAsync(Guid id)
```

Comment:
please add CancellationToken param for this method, and for according repository method